AMBuildScript

```diff
@@ -76,6 +76,7 @@ PossibleSDKs = {
   'contagion':  SDK('HL2SDKCONTAGION', '2.contagion', '14', 'CONTAGION', WinOnly, 'contagion'),
   'bms':  SDK('HL2SDKBMS', '2.bms', '10', 'BMS', WinLinux, 'bms'),
   'mock': SDK('HL2SDK-MOCK', '2.mock', '999', 'MOCK', Mock, 'mock'),
+  'pvkii': SDK('HL2SDKPVKII', '2.pvkii', '23', 'PVKII', WinLinux, 'pvkii'),

```

Comment:
Unlike the MMBackend and SOURCE_ENGINE_BLAH defines in ISmmPluginExt.h, the order of these ones have significance, used for the various "SE_" defines in the codebase. There are some comparison checks we use in places against certain engine baselines, like "orangebox" (tf/css/sdk2013/etc), l4d, l4d2, portal2, etc. (These checks can't always be perfect due to the engine branches not being linear, but some code relies on this)

I think we should use 10 for pvkii, and shift the greater ones forward by 1.

Reply:
Yeah, it felt a little awkward with PVKII at 23 and then having deal with all the >= preprocessor checks, I can look at refactoring and testing that out.